addons/website/views/snippets/s_banner_connected.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_banner_connected" name="Banner connected">
+    <section
+        class="s_banner_connected o_cc o_cc5 oe_img_bg pt128 pb136 parallax"
+        data-oe-shape-data="{'shape':'web_editor/Connections/13','colors':{'c5':'#FFFFFF'},'flip':[],'showOnMobile':false,'shapeAnimationSpeed':'0'}"
```

Comment:
Also, if you don't use any flip or shape animation, I think you can remove the options. Not 100% sure tho, you should try 😄

```suggestion
        data-oe-shape-data="{'shape':'web_editor/Connections/13','colors':{'c5':'#FFFFFF'},'showOnMobile':false}"
```